tools/accuracy_checker/accuracy_checker/metrics/mpjpe_multiperson.py

```diff
@@ -0,0 +1,113 @@
+            prediction.y_3d_values[pose_id] += prediction.translations[pose_id, 1]
+            prediction.z_3d_values[pose_id] += prediction.translations[pose_id, 2] * annotation.fx
+
+        # since pelvis does not detected, remove it before evaluation
+        kpt_mask = np.ones_like(annotation.x_3d_values[0], dtype=bool)
+        kpt_mask[2] = False
+        annotation.x_3d_values = annotation.x_3d_values[:, kpt_mask]
+        annotation.y_3d_values = annotation.y_3d_values[:, kpt_mask]
+        annotation.z_3d_values = annotation.z_3d_values[:, kpt_mask]
+        if prediction.size:
+            prediction.x_3d_values = prediction.x_3d_values[:, kpt_mask]
+            prediction.y_3d_values = prediction.y_3d_values[:, kpt_mask]
+            prediction.z_3d_values = prediction.z_3d_values[:, kpt_mask]
+
+        matching_result = {
+            'gt_coordinates': [],
+            'predicted_coordinates': [],
+            'gt_size': annotation.size
+        }
+        sorted_prediction_ids = np.argsort(-prediction.scores, kind='stable')
+        mask = np.ones(annotation.size, dtype=bool)
+        for prediction_id in range(prediction.size):
+            max_iou = 0
+            matched_id = -1
+            bbox = prediction.bboxes[sorted_prediction_ids[prediction_id]]
+            for annotation_id in range(annotation.size):
+                if not mask[annotation_id]:
+                    continue
+                iou = _get_iou(bbox, annotation.bboxes[annotation_id])
+                if iou > max_iou:
+                    max_iou = iou
+                    matched_id = annotation_id
+            if matched_id >= 0:
+                mask[matched_id] = 0
+                matching_result['gt_coordinates'].append(np.transpose(np.array(
+                    [annotation.x_3d_values[matched_id], annotation.y_3d_values[matched_id],
+                     annotation.z_3d_values[matched_id]]), (1, 0)))
+                matching_result['predicted_coordinates'].append(np.transpose(np.array(
+                    [prediction.x_3d_values[sorted_prediction_ids[prediction_id]],
+                     prediction.y_3d_values[sorted_prediction_ids[prediction_id]],
+                     prediction.z_3d_values[sorted_prediction_ids[prediction_id]]]), (1, 0)))
+        self.matching_results.append(matching_result)
+
+    def evaluate(self, annotations, predictions):
+        total_mpjpe = 0
+        for matching_result in self.matching_results:
+            image_mpjpe = 0
+            for input, target in zip(matching_result['predicted_coordinates'],
+                                     matching_result['gt_coordinates']):
+                image_mpjpe += _mpjpe(input, target)
+
+            if len(matching_result['gt_coordinates']):
+                image_mpjpe /= len(matching_result['gt_coordinates'])
+            total_mpjpe += image_mpjpe
+        if len(self.matching_results):
+            total_mpjpe /= len(self.matching_results)
+        return total_mpjpe
```

Comment:
if it is error rate, I suggest to add some meta info about result in metric meta for correct result formatting
```python
self.meta.update({
    'scale': 1,
    'postfix': `sm`
})
```
It allows to get 10 sm instead 1000% as metric result